cg/store/crud/read.py

```diff
@@ -187,6 +194,19 @@ def get_analysis_by_case_entry_id_and_completed_at(
             completed_at_date=completed_at_date,
         ).first()
 
+    def get_latest_started_analysis_for_case(self, case_id: str) -> Analysis:
+        """Return the latest started analysis for a case.
+        Raises:
+            AnalysisDoesNotExistError if no analysis is found.
+        """
+        case: Case = self.get_case_by_internal_id(case_id)
```

Comment:
```suggestion
        case: Case | None = self.get_case_by_internal_id(case_id)
```

Until we get that version that always returns an item or raises :)